src/sql/category_by_name.sql

```diff
@@ -0,0 +1,4 @@
+SELECT name
```

Comment:
You might want to also select `id` for the commands that will be modifying the category. But that can be done later, as it becomes necessary.